src/Preview.js

```diff
@@ -99,6 +106,7 @@ export default class Preview extends Component {
         onKeyPress={this.handleKeyPress}
       >
         {playIcon || defaultPlayIcon}
+        {previewImage.isComponent && previewImage.component}
       </div>
```

Comment:
If you switch the order of these you don't need to add the z-index to the shadow style.
i.e 
```
{previewImage.isComponent && previewImage.component}
{playIcon || defaultPlayIcon}
```